examples/view_image.py

```diff
@@ -0,0 +1,23 @@
+"""
+View Image and Launch napari
```

Comment:
```suggestion
Launch napari to view an image
```
I think this order is clearer?

Reply:
I agree its clearer, will commit after other review

Reply:
I agree also.